api/napi/src/types/brush.rs

```diff
@@ -151,4 +161,9 @@ impl JsBrush {
     pub fn brush(&self) -> External<Brush> {
         External::new(self.inner.clone())
     }
+
+    #[napi]
+    pub fn to_string(&self) -> String {
+        self.color().to_string()
```

Comment:
I suggest to add a comment here that this is wrong for brushes that aren't solid colors. Evidently a gradient brush should not convert to a string like this. Alternatively, perhaps an exception should be thrown, that this is not implemented yet?